api/routes/employees.js

```diff
@@ -3,26 +3,47 @@
  */
 
 const express = require('express');
-const Sequelize = require('sequelize');
 const router = express.Router();
 
+const logger = require('../logger');
 const Employee = require('../models/employee');
+const db = require('../db');
+const queries = require('../db/queries');
 
 router.get('/', async (req, res, next) => {
-    const employees = await Employee.findAll();
-    res.send(employees);
-});
-router.get('/statuses', async (req, res, next) => {
-    res.send([]);
+    try {
+        const employees = await Employee.findAll();
+        res.json(employees);
+    } catch(err) {
+        res.status(500).send(err.message);
+        logger.error('Error employee get', err);
+    }
 });
+
 router.post('/', async (req, res, next) => {
-    let employees = req.body,
+    try {
+        let employees = req.body,
         upsertPromises = [];
-    employees.forEach(employee => {
-        upsertPromises.push(Employee.upsert(employee));
-    });
-    let result = await upsertPromises;
-    res.send('Employees are successfully inserted!');
+
+        employees.forEach(employee => {
+            upsertPromises.push(Employee.upsert(employee));
+        });
+        let result = await Promise.all(upsertPromises);
+        res.send('Employees are successfully inserted!');
+    } catch(err) {
+        res.status(500).send(err.message);
+        logger.error('Error employee post', err);
+    }
+});
+
+router.get('/statuses', async (req, res, next) => {
+    try {
+        const employees = await db.query(queries.getStatuses, { type: db.QueryTypes.SELECT});
+        res.json(employees);
+    } catch (err) {
+        res.status(500).send(err.message);
+        logger.error('Error employee post', err);
```

Comment:
please change to a human readable sentence: "An error occurred when getting employee statuses"